app/views/main.scala.html

```diff
@@ -15,13 +17,32 @@
         <link rel="shortcut icon" type="image/png" href="@routes.Assets.versioned("images/favicon.png")">
         <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
         <script src="@routes.Assets.versioned("javascripts/main.js")" type="text/javascript"></script>
+        <script type="text/javascript">
+            window.guardian = {
+                'urls': {
```

Comment:
Previously these urls were hardcoded into the client for CODE + PROD.
Now, the client receives the correct urls for the stage in `window.guardian.urls`